tensorflow_addons/optimizers/lamb_test.py

```diff
@@ -401,3 +401,11 @@ def test_get_config(self):
         opt = lamb.LAMB(1e-4)
         config = opt.get_config()
         self.assertEqual(config["learning_rate"], 1e-4)
+
+    def test_exclude_weight_decay(self):
```

Comment:
Could you add a test for `exclude_from_layer_adaption` as well please?

Reply:
Done